ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/dropdown.view.js

```diff
@@ -101,7 +102,7 @@ define([
             return this.el;
         },
         determineSelections: function () {
-            var values = this.model.get('value');
+            var values = Common.getArrayFromValue(this.model.get('value'));
```

Comment:
❓ I thought we're trying to use es6 syntax more, no? This might be better served with a `const`